Best Practices/Design Patterns/Behavioral Patterns.md

```diff
@@ -135,43 +135,43 @@ With the Chain of Responsibility pattern, we avoid coupling the sender of a requ
 To implement this pattern we create a chain of objects to examine requests. Each object in turn examines a request and either handles it or passes it on to the next object in the chain. A client that sends a request doesn’t have to know the chain’s structure nor keep direct references to its members.
 
 ```c#
-abstract class CarService
+abstract class CarMaintenanceService
```

Comment:
While we're renaming things here, I have a suggestion for a small improvement. Since the two implementations below are services for maintaining a specific part of a car, maybe it would be more appropriate to name this base class accordingly - the current name suggests that the service implementation will maintain the whole car, which is not the case.

Reply:
Would something like `CarPartMaintenanceService` suffice?